cfme/utils/appliance/__init__.py

```diff
@@ -2443,6 +2443,18 @@ def provider_based_collection(self, provider, coll_type='vms'):
             raise ValueError('No support for coll_type: "{}" collection name lookup'
                              .format(coll_type))
 
+    def enable_migration_ui(self):
+        self.update_advanced_settings({'product': {'transformation': True}})
+        self.restart_evm_service()
+        self.wait_for_evm_service()
+        self.wait_for_web_ui()
+
+    def disable_migration_ui(self):
+        self.update_advanced_settings({'product': {'transformation': False}})
+        self.restart_evm_service()
+        self.wait_for_evm_service()
+        self.wait_for_web_ui()
```

Comment:
**REQUIRED**
I guess this is only for 5.9.
```python
def _switch_migration_ui(self,  enable):
    self.update_advanced_settings({'product': {'transformation': enable}})
    self.restart_evm_service()
    self.wait_for_evm_service()
    self.wait_for_web_ui()

def enable_migration_ui(self)
    self._switch_migration_ui(self,  True)

def disable_migration_ui(self)
   self. _switch_migration_ui(self,  False)
```

Reply:
made the changes.